packages/itinerary-body/src/TransitLegBody/view-trip-button.tsx

```diff
@@ -28,7 +28,7 @@ class ViewTripButton extends Component<Props> {
 
   render(): ReactElement {
     return (
-      <S.ViewerButton onClick={this.onClick} type="button">
+      <S.ViewerButton href="#" onClick={this.onClick} tabIndex={0}>
```

Comment:
`href="#"` does not work, it will clear out the query params in the url bar and that will mess up OTP-RR state, so i guess if we must use `<a>`, we are stuck with not setting `href` and keep `tabIndex={0}`.

Reply:
@amy-corson-ibigroup Do this ^^